Makefile

```diff
@@ -32,3 +34,18 @@ release: b2d_iso get_gh-release
 	cp $(BUILD_DIR)/minishift-b2d.iso release/
 	$(BUILD_DIR)/gh-release checksums sha256
 	$(BUILD_DIR)/gh-release create minishift/minishift-b2d-iso $(VERSION) master v$(VERSION)
+
+.PHONY: prepare_for_test
```

Comment:
This is actually not a phony target. This is very much a target which you want to skip if the binary is there